packages/techdocs-common/src/stages/generate/helpers.ts

```diff
@@ -298,6 +266,87 @@ export const patchMkdocsYmlPreBuild = async (
   }
 };
 
+/**
```

Comment:
Wonder if it makes sense to separate out these yml (and perhaps even `techdocs_metadata.json` transformations out of `helpers` and into like a...  Transformer file or folder?  😐

Reply:
I moved these now to initially reside in mkdocs specific 'patchers' file, since that's what they are called.